Extension/c_cpp_properties.schema.json

```diff
@@ -142,6 +142,16 @@
                             }
                         },
                         "additionalProperties": false
+                    },
+                    "properties": {
```

Comment:
The name "properties" is too generic and can be ambiguous when referring to the other properties in c_cpp_properties.json. Can this be changed to a more distinct name?

Reply:
Maybe "customProperties"?

Reply:
happy to change the name, a few other suggestions include "userProperties", "variantProperties". I have a slight preference for "userProperties" or "customProperties"

Reply:
sounds good. The method is called "activeConfigProperty", does "activeConfigCustomVariable" sound sane?

Reply:
"activeConfigCustomVariable" sounds good for the method.